scripts/lint_yaml.py

```diff
@@ -412,6 +470,8 @@ def validate_person(self, person, filename, person_type, date=None):
         uid = person["id"].split("/")[1]
         if uid not in filename:
             self.errors[filename].append(f"id piece {uid} not in filename")
+        if not names_consistent(person):
+            self.errors[filename].append(f"inconsistent name")
```

Comment:
gotcha, misunderstood what this was doing based on some of the other conversations.  makes sense-- but can we make this error message a bit more clear (maybe include the two names?)

Reply:
1bc0f74f6ae566bd7489f1daa8ad58e35b73e097 (include mismatched name values for lint 'inconsistent-names' errors)